src/widget/weffectchainpresetbutton.cpp

```diff
@@ -43,13 +43,15 @@ void WEffectChainPresetButton::setup(const QDomNode& node, const SkinContext& co
                 this,
                 &WEffectChainPresetButton::populateMenu);
     }
+    m_pMenu->setToolTipsVisible(true);
     populateMenu();
 }
 
 void WEffectChainPresetButton::populateMenu() {
     m_pMenu->clear();
 
     // Chain preset items
+    const auto bem = m_pEffectsManager->getBackendManager();
```

Comment:
```suggestion
    const auto* pBem = m_pEffectsManager->getBackendManager();
```

Reply:
Else the pointer is const, not the object it is pointing to.
Below the same.

Reply:
This returns `EffectsBackendManagerPointer`,  I don't understand how EffectsBackendManager could be const here.
Now it's explicit `const EffectsBackendManagerPointer` instead of `const auto`.

Reply:
Oh, yes I missed that. Thank you.